src/test/resources/json-merge-test/json-merge-test20.json

```diff
@@ -0,0 +1,5 @@
+[
+  [],
+  [],
+  []
+]
```

Comment:
This is good, could you please move it to [json-merge-test/edge-cases](https://github.com/hemantsonu20/json-merge/blob/master/src/test/resources/json-merge-test/edge-cases/)